app/controllers/finders_controller.rb

```diff
@@ -27,7 +27,7 @@ def show
           redirect_to_destination
         else
           @search_query = initialize_search_query(is_for_feed: true)
-          expires_in(ATOM_FEED_MAX_AGE, public: true)
+          expires_in(content_item.max_age, public: content_item.cache_public)
```

Comment:
You no longer need this line since there's a `before_action` that does the same thing.

Reply:
ok, I'll remove